nvflare/app_common/storages/filesystem_storage.py

```diff
@@ -72,6 +76,24 @@ def _write(path: str, content, mv_file=True):
         os.rename(tmp_path, path)
 
 
+def _write_multi(output_zip_file_name: str, content: List[str]):
+    with tempfile.TemporaryDirectory() as td:
+        for c in content:
+            if os.path.isfile(c):
+                basename = os.path.basename(c)
```

Comment:
Should we keep the folder structure of the content? For example, if content has 'folder1/file', 'folder2/file', this would end up with one file.